src/dataExplorer/components/Schema.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {FC, useContext, useMemo} from 'react'
+import {useSelector} from 'react-redux'
+
+// Components
+import {DapperScrollbars} from '@influxdata/clockface'
+import SearchWidget from 'src/shared/components/search_widget/SearchWidget'
+import BucketSelector from 'src/dataExplorer/components/BucketSelector'
+import MeasurementSelector from 'src/dataExplorer/components/MeasurementSelector'
+import FieldSelector from 'src/dataExplorer/components/FieldSelector'
+import TagSelector from 'src/dataExplorer/components/TagSelector'
+
+// Context
+import {
+  NewDataExplorerContext,
+  NewDataExplorerProvider,
+} from 'src/dataExplorer/context/newDataExplorer'
+import QueryProvider from 'src/shared/contexts/query'
+import {BucketProvider} from 'src/shared/contexts/buckets'
+import {MeasurementProvider} from 'src/dataExplorer/context/measurements'
+import {FieldsProvider} from 'src/dataExplorer/context/fields'
+import {TagsProvider} from 'src/dataExplorer/context/tags'
+
+// Types
+import {QueryScope} from 'src/types'
+
+// Utils
+import {getOrg} from 'src/organizations/selectors'
+
+// Style
+import './Schema.scss'
+
+const FieldsTags: FC = () => {
+  const {
+    selectedBucket,
+    selectedMeasurement,
+    searchTerm,
+    setSearchTerm,
+  } = useContext(NewDataExplorerContext)
+
+  const handleSearchFieldsTags = (searchTerm: string): void => {
+    setSearchTerm(searchTerm)
+  }
+
+  return useMemo(() => {
+    if (!selectedBucket || !selectedMeasurement) {
+      return null
+    }
+
+    return (
+      <div className="container-side-bar">
+        <SearchWidget
+          placeholderText="Search fields and tags..."
```

Comment:
There’s a PR up to remove ellipses, did we want them here?

Reply:
Nice catch!